AutomatticTracks/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 
 repositories {
     google()
-    jcenter()
+    mavenCentral()
```

Comment:
This is already specified in project's `build.gradle` ([here](https://github.com/Automattic/Automattic-Tracks-Android/blob/develop/build.gradle#L4)) so it would work without this line. 

I added `mavenCentral()` explicitly as `Sentry` is published in this repository. I wanted to provide all needed repositories for a module.